install/running_install.adoc

```diff
@@ -403,6 +403,32 @@ ifdef::openshift-origin[]
 endif::[]
 ----
 
+[running-the-advanced-installation-openstack]
+==== Running the Installation Playbook for OpenStack
+
+[IMPORTANT]
+====
+The OpenStack installation playbook is a Technology Preview feature.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
```

Comment:
They are supported however they get a lower priority. I would stick to language we use about TP in other places (copy a blurb from one of those). 

It looks like we're simply saying: `X is in Technology Preview and is not for production workloads.` in the release notes now. ` and referring to the link your providing in 419/420

Reply:
@sferich888, I did take that language from another TP note, and it matches 6 other random notes I checked. Will you link me to an example of what you were expecting?